pt-BR/mail/chrome/messenger/offline.properties

```diff
@@ -5,16 +5,16 @@ downloadMessagesCheckboxLabel1=Sempre perguntar ao entrar no modo offline
 
 # Send Messages Prompt
 sendMessagesWindowTitle1=Mensagens da pasta “Saída”
-sendMessagesLabel1=Do you want to send the messages in your Unsent Messages folder?
+sendMessagesLabel1=Deseja reenviar as mensagens na sua pasta Não enviados?
```

Comment:
CORRECTIONS NEEDED it should be sendMessagesLabel1=Deseja enviar as mensagens na sua pasta Não enviados?